recipes/pgcgap/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pgcgap" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "71636dd95367d8879fa8d030f871db71be539aa01905701567479ea2b1d89a45" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/liaochenlanruo/pgcgap/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl
+    - perl-bioperl >=1.7.2
+
+  run:
+    - abyss >=2.1.5
+    - prokka >=1.13.4
+    - cd-hit >=4.8.1
+    - mafft >=7.402
+    - fasttree >=2.1.10
+    - roary >=3.12.0
+    - orthofinder >=2.2.7
+    - diamond >=0.9.24
+    - mmseqs2 >=7-4e23d
+    - fastme >=2.1.5
+    - fastani >=1.1
+    - perl-file-tee >=0.07
+    - perl-data-dumper >=2.173
+    - perl-pod-usage >=1.69
+    - perl-getopt-long >=2.50
+    - htslib >=1.9
+    - snippy >=4.3.6
+    - sickle-trim >=1.33
+    - gubbins >=2.3.4
+    - r-ggplot2
+    - r-corrplot
+    - matplotlib
+    - seaborn
+    - numpy
+    - pandas
+    - biopython
+    - r-plotrix
+
+
+test:
+  commands:
+    - pgcgap --check-external-programs
+    - pgcgap --version
+    - pgcgap --help
+
+
+about:
+  home: https://github.com/liaochenlanruo/pgcgap/blob/master/README.md
+  license: GPLv3
+  license_family: GPL
```

Comment:
Does this package happen to ship a copy of its license? If so, please add the path to it in `license_file:`